### PR TITLE
Move fx2trt out of PyTorch core (#740)

### PR DESCRIPTION
**Pull request by yinghai**
Summary:
Pull Request resolved: https://github.com/pytorch/benchmark/pull/740

To fx2trt out of tree to remove bloatness of PyTorch core.

It's the first and major step. Next, we will move acc_tracer out of the tree and rearrange some fx passes.

Differential Revision: D34065866

### CODE REVIEW

File: test/fx2trt/core/test_import_fx2trt.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 # Test that this import should not trigger any error when run
 # in non-GPU hosts, or in any build mode.
-import torch.fx.experimental.fx2trt.lower as fxl  # noqa: F401
+import fx2trt_oss.fx.lower as fxl  # noqa: F401
```
Comment (suo) [inline test/fx2trt/core/test_import_fx2trt.py:8]: For my understanding—is the stuff in `test/fx2trt` eventually supposed to go as well?

**Review by suo — APPROVED** [review]

**Comment by suo** [comment]
One small note, @wushirong added tensorrt to the PyTorch OSS CI docker images I believe. Are those removable after fx2trt moves out of core?

File: test/fx2trt/trt_lower/test_diagnostics.py [hunk 2]
```diff
@@ -0,0 +1,184 @@
+# Owner(s): ["oncall: aiacc"]
```
Comment (suo) [inline test/fx2trt/trt_lower/test_diagnostics.py:1]: is this file supposed to be added?

Comment (yinghai) [reply test/fx2trt/trt_lower/test_diagnostics.py]: Sorry this is the file that this diff was rebased on. Should be gone already.

**Comment by yinghai** [comment]
> One small note, @wushirong added tensorrt to the PyTorch OSS CI docker images I believe. Are those removable after fx2trt moves out of core?

Yeah. I'm asking @wushirong to remove those.

File: test/fx2trt/core/test_import_fx2trt.py (hunk 1 above)
Comment (yinghai) [reply test/fx2trt/core/test_import_fx2trt.py:8]: Yes. They will need to go. I'm actually surprised that they are here.

**Pull request closed without merging**